### PR TITLE
Hibernate envers for db versioning

### PR DESCRIPTION
**Pull request by honest-niceman**

### CODE REVIEW

File: database-versioning/database-versioning.md [hunk 1]
```diff
@@ -728,59 +742,11 @@ Each DB-agnostic type has a set of aliases (for example, "java.sql.Types.VARCHAR
...
-    REVTYPE tinyint,
-    created_on timestamp,
-    firstName varchar(255),
-    lastName varchar(255),
-    primary key (id, REV)
-)
-```
-
-When generating migration scripts, JPA Buddy will understand that the `Customer_AUD` table doesn't have to have a corresponding JPA entity, and will not generate a drop statement for it.
+Hibernate Envers offers a lot of customization options. You can specify the desired configuration in the JPA Buddy settings (2), or let JPA Buddy read the existing settings from `.properties` files automatically (1).
```
Comment (AmirShelli) [inline database-versioning/database-versioning.md:749]:
```suggestion
Hibernate Envers offers various customization options. You can specify the desired configuration in the JPA Buddy settings (2), or let JPA Buddy read the existing settings from `.properties` files automatically (1).
```

File: database-versioning/database-versioning.md [hunk 2]
```diff
@@ -570,6 +570,20 @@ Before Hibernate 6, the Hibernate Types library or `@Type`/`@Converter` annotati
...
+This functionality becomes available only if the <a href="https://jpa-buddy.com/documentation/#dependencies" target="_blank">corresponding dependency</a> is added to the project.
+</div>
+
+<a href="https://hibernate.org/orm/envers/" target="_blank">Hibernate Envers</a> is a module that enables easy auditing of entities in a database. JPA Buddy can generate all the required tables for Hibernate Envers to function correctly. This includes audit tables for entities marked with the `@Audited` annotation and revision table for the entity annotated with `@RevisionEntity` annotation. Check out how it works in action:
+
+<div class="youtube">
+<iframe width="560" height="315" src="https://www.youtube.com/embed/k5OTgQBmSrg" title="YouTube video player" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+</div>
+
+JPA Buddy offers flexible settings for Hibernate Envers. You can find more information about these settings in the [relevant section](#hibernate-envers).
```
Comment (AmirShelli) [inline database-versioning/database-versioning.md:585]:
```suggestion
JPA Buddy offers flexible settings for Hibernate Envers. For more details, refer to the [corresponding section](#hibernate-envers).
```

File: installation-guide/installation-guide.md [hunk 3]
```diff
@@ -226,6 +226,15 @@ Since JPA Buddy supports Hibernate, EclipseLink, SpringData, MapStruct, Liquibas
     <td class="tg-0lax"><a href="https://mvnrepository.com/artifact/com.blazebit/blaze-persistence-integration-spring-data-base" target="_blank" rel="noopener noreferrer">Blazebit Persistence Integration Spring Data Base</a></td>
     <td class="tg-0lax">- Create Spring Data JPA repository for Blaze Persistence Entity View.</td>
   </tr>
+  <tr>
+    <td class="tg-0lax"><a href="https://mvnrepository.com/artifact/org.hibernate/hibernate-envers" target="_blank" rel="noopener noreferrer">Hibernate Envers</a></td>
+    <td class="tg-0lax">
+    - Generate database migration scripts for audit tables.<br>
+    - Create Spring Data JPA revision repositories.<br>
+    - Mark entities and its fields that need to be audited via JPA Designer. <br>
```
Comment (AmirShelli) [inline installation-guide/installation-guide.md:234]: maybe "Annotate entities and their fields for auditing via JPA Designer."?

Comment (honest-niceman) [inline installation-guide/installation-guide.md:234]:
```suggestion
    - Annotate entities and their fields for auditing via JPA Designer. <br>
```

**Pull request merged**